### ISSUE TITLE
undesirable_operator_linter should accept := as input

### ISSUE THREAD

**Issue by MichaelChirico** [r-lib/lintr#1909]
Seems to me `:=` is as valid an operator as any since it's typically used in infix format, but it's currently not supported:

```r
lintr::lint("\na := 2", lintr::undesirable_operator_linter(c(`:=` = NA)))
# Error in lintr::undesirable_operator_linter(c(`:=` = NA)) : 
#   Did not recognize any valid operators in request for: :=
```

**Comment by MichaelChirico** [comment]
This is already fixed, I guess by #2115 

```r
lintr::lint("\na := 2", lintr::undesirable_operator_linter(c(`:=` = NA)))
# <text>:2:3: warning: [undesirable_operator_linter] Avoid undesirable operator `:=`.
# a := 2
#   ^~
```
